slack/src/main/java/org/togglz/slack/SlackWrapperStateRepository.java

```diff
@@ -0,0 +1,16 @@
+package org.togglz.slack;
+
+import org.togglz.core.repository.StateRepository;
+import org.togglz.core.repository.composite.CompositeStateRepository;
+
+/**
+ * @author Tomasz Skowroński
+ * @since 2.4.0
+ */
+public class SlackWrapperStateRepository extends CompositeStateRepository {
```

Comment:
Yeah, that's what I was thinking about. IMO this makes much more sense, doesn't it? 

And it reduces the coupling to the StateRepository SPI. Which is a good thing because we may think about adding more explicit support for notifications to the core API at some point.